src/olympia/reviewers/utils.py

```diff
@@ -1360,6 +1373,61 @@ def reject_multiple_versions(self):
                 content_review=self.content_review,
             )
 
+    def unreject_multiple_versions(self):
+        """Un-reject a list of versions."""
+        # self.version and self.file won't point to the versions we want to
+        # modify in this action, so set them to None before finding the right
+        # versions.
+        latest_version = self.version
+        self.version = None
+        self.file = None
+        now = datetime.now()
+        # we're only supporting non-automated reviews right now:
+        assert self.human_review
+
+        log.info(
+            'Making %s versions %s awaiting review (not disabled)'
+            % (self.addon, ', '.join(str(v.pk) for v in self.data['versions']))
+        )
+
+        for version in self.data['versions']:
+            self.set_file(amo.STATUS_AWAITING_REVIEW, version.file)
+
+            self.log_action(
+                action=amo.LOG.UNREJECT_VERSION,
+                version=version,
+                file=version.file,
+                timestamp=now,
+                user=self.user,
+            )
+
+        # if these are listed versions then the addon status may need updating
+        if (
+            self.data['versions']
+            and latest_version.channel == amo.CHANNEL_LISTED
+            and self.addon.status == amo.STATUS_NULL
+        ):
+            log.info(
+                'Changing add-on status [%s]: %s => %s (%s).'
+                % (
+                    self.addon.id,
+                    self.addon.status,
+                    amo.STATUS_NOMINATED,
+                    'unrejecting versions',
+                )
+            )
+            self.addon.update(status=amo.STATUS_NOMINATED)
+            ActivityLog.create(
+                amo.LOG.CHANGE_STATUS, self.addon, self.addon.status, user=self.user
+            )
+            self.addon.update_version()
+
+        # The reviewer should be automatically subscribed to any new versions posted to
```

Comment:
I am not sure we need to subscribe the reviewer. What scenario did you have in mind?

Reply:
🤷 up to you really.  It's standard for the other reviewer actions but we can omit it for this one.

Reply:
https://github.com/mozilla/addons-server/pull/19728/commits/a9a7883bbaed4f0a26b2954cae6d94c85a333341

Reply:
Yes, let's omit here. Putting it back into `AWAITING_REVIEW` is usually followed by another action (primarily _Approve_), so there isn't much of a need to subscribe reviewers automatically.